althea_kernel_interface/src/exit_client_tunnel.rs

```diff
@@ -66,7 +65,7 @@ impl KernelInterface {
     }
 
     pub fn set_route_to_tunnel(&self, gateway: &IpAddr) -> Result<(), Error> {
-        self.run_command("ip", &["route", "del", "default"]);
+        self.run_command("ip", &["route", "del", "default"]).expect("Failed to delete default route");
```

Comment:
we might not have a default route to delete, but we should be a bit more explicit about ignoring errors